runtime/testnet/src/filters.rs

```diff
@@ -19,19 +19,15 @@ use super::*;
 pub struct TestnetCallFilter;
 impl Contains<RuntimeCall> for TestnetCallFilter {
 	fn contains(call: &RuntimeCall) -> bool {
-		let is_core_call = matches!(call, RuntimeCall::System(_) | RuntimeCall::Timestamp(_));
+		let is_core_call = matches!(
+			call,
+			RuntimeCall::System(_) | RuntimeCall::Timestamp(_) | RuntimeCall::TxPause(_)
+		);
 		if is_core_call {
 			// always allow core call
 			return true;
 		}
 
-		let is_paused =
-			pallet_transaction_pause::PausedTransactionFilter::<Runtime>::contains(call);
-		if is_paused {
-			// no paused call
-			return false;
-		}
```

Comment:
This is still needed, needs to be replaced with 
https://github.com/paritytech/polkadot-sdk/blob/60ac5a723c6c0eeef0e35a0788792378c8200ec5/substrate/frame/tx-pause/src/lib.rs#L293

without it the calls arent actually blocked